src/many-kvstore/tests/kvstore.rs

```diff
@@ -0,0 +1,75 @@
+pub mod common;
+use crate::common::{setup, Setup};
+use minicbor::bytes::ByteVec;
+
+use many::{
+    server::module::kvstore::{
+        KvStoreModuleBackend, 
+        KvStoreCommandsModuleBackend, 
+        InfoArg,
+        PutArgs,
+        GetArgs,
+        DeleteArgs
+    },
+};
+
+#[test]
+fn kvstore_info() {
+    let Setup {
+        module_impl, 
+        id, 
+        ..
+    } = setup();
+    
+    let result = module_impl.info(&id, InfoArg {});
```

Comment:
You should somehow test that the return value makes sense.

Reply:
Updated. The hash may vary so I went with a length check.